apps/clickhouse/24.11.1-alpine/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
     volumes:
       - ./clickhouse:/var/lib/clickhouse/
       - ./clickhouse-server:/var/log/clickhouse-server/
-    image: clickhouse/clickhouse-server:24.11.1-alpine
+    image: clickhouse/clickhouse-server:24.11.2-alpine
     ulimits:
       nofile:
         soft: 262144
```

Comment:
1. Correct version number from "24.11.1-alpine" to "24.11.2-alpine".
   Original change:
   ```
      - clickhouse/clickhouse-server:24.11.1-alpine
   ```

   Change to:
   ```
      - clickhouse/clickhouse-server:24.11.2-alpine
   ```